### DISCUSSION TITLE
Running on QEMU x86

### DISCUSSION THREAD

**Discussion by antaradey-itk** [eclipse-leda/leda-distro#68] [Q&A] [answered]
Hello.

I have been trying to launch leda on qemu x86. So far, I have installed Ubuntu 22.04 on Virtual box 7.0.6 (Windows 10 host). 
I have completed installing qemu and unzipping the respective archive for leda. 

Now when I try to run the shell script for execution, it throws the following error----

**qemu-system-x86_64: -drive if=pflash,format=qcow2,file=ovmf.qcow2: Could not open 'ovmf.qcow2': No such file or directory**

Has anyone encountered this issue before?

Any help will be appreciated. Thanks 👍

**Comment by mikehaller** [comment]
The file ovmf.qcow2 is included in the release archive.
Are you running the ./run-leda.sh script in the current working directory?

If you start it from a different location, it won't work currently, as the scripts expect to be run in the same directory as the needed .qcow2 files.

Did you also check if there is maybe a problem with file permissions? Unlikely, but worth checking.

**Reply by antaradey-itk** [reply]
Ok. So the problem was during extraction. The file did not get extracted using tar -xf command, and threw 'unexpected EOL error', - not sure of the issue here. 

I made the extraction using GUI, and it works fine now, except it asks for a **qemux86-64 login** after boot up.

**Reply by vasilvas99** [reply, marked as answer]
Can you try `root` as user and no password?

**Reply by antaradey-itk** [reply]
It works. Thanks
